### PR TITLE
break out deploy configs and add a 3 testnet script

### PR DESCRIPTION
**Pull request by prestwich**
- restructure deploy configs and scripts
- add a 3 testnet deploy script

### CODE REVIEW

File: typescript/optics-deploy/config/testnets/rinkeby.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const chainJson: ChainJson = {
+  name: 'rinkeby',
+  rpc,
+  deployerKey: process.env.RINKEBY_DEPLOYER_KEY,
+  domain: 2000,
+};
+
+export const chain = toChain(chainJson);
+
+export const devConfig: CoreConfig = {
```
Comment (ErinHales) [inline typescript/optics-deploy/config/testnets/rinkeby.ts:24]: Move to an `index.ts`?

File: typescript/optics-deploy/scripts/threeTestnets/core.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+let kovanConfig =
+  environment === 'staging' ? kovan.stagingConfig : kovan.devConfig;
+let rinkebyConfig =
+  environment === 'staging' ? rinkeby.stagingConfig : rinkeby.devConfig;
+
+const alfaDeploy = new CoreDeploy(alfajores.chain, alfaConfig);
+const kovanDeploy = new CoreDeploy(kovan.chain, kovanConfig);
+const rinkebyDeploy = new CoreDeploy(rinkeby.chain, rinkebyConfig);
+
+deployNChains([alfaDeploy, kovanDeploy, rinkebyDeploy]);
```
Comment (ErinHales) [inline typescript/optics-deploy/scripts/threeTestnets/core.ts:21]: Have you tested these deploys yet?

File: typescript/optics-deploy/config/testnets/rinkeby.ts (hunk 1 above)
Comment (ErinHales) [reply typescript/optics-deploy/config/testnets/rinkeby.ts:24]: Plus `stagingConfig` and `bridgeConfig`

File: typescript/optics-deploy/scripts/threeTestnets/core.ts (hunk 2 above)
Comment (prestwich) [reply typescript/optics-deploy/scripts/threeTestnets/core.ts:21]: no, working on it

File: typescript/optics-deploy/config/testnets/rinkeby.ts (hunk 1 above)
Comment (prestwich) [reply typescript/optics-deploy/config/testnets/rinkeby.ts:24]: have `testnets/index.ts` export all the configs?

Comment (ErinHales) [reply typescript/optics-deploy/config/testnets/rinkeby.ts:24]: Oh, never mind.  The way this showed up in the file diff was confusing.  I thought we had a single `devConfig`/`stagingConfig` that we were using for all three deploys

**Review by ErinHales — APPROVED** [review]
Looks good!

**Comment by prestwich** [comment]
will merge as soon as I have a reasonable deploy and see config output

**Pull request merged**